src/main/kotlin/step3/racingcar/utils/CarGenerator.kt

```diff
@@ -0,0 +1,13 @@
+package step3.racingcar.utils
+
+import step3.racingcar.domain.Car
+
+object CarGenerator {
+    fun generate(carNames: List<String>): MutableList<Car> {
+        val cars = mutableListOf<Car>()
+        carNames.forEach {
+            cars.add(Car(it))
+        }
+        return cars
```

Comment:
아래처럼 확장함수를 이용해볼 수 있겠네요! :)
```suggestion
        return carNames.map { Car(it) }
```